### PR TITLE
virtio_console_guest:update code format to keep compatibility

### PR DESCRIPTION
**Pull request by nanliu-r**
ID: 1427

Drop the 'six' dependency for making the tool used standalone.

### CODE REVIEW

**Comment by nanliu-r** [comment]
Test PASS with both RHEL vm(default python version) and Windows vm (Python2).
@luckyh Hi, could you help review this? Thanks

File: virttest/shared/scripts/virtio_console_guest.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
-#!/usr/bin/python
 # -*- coding: utf-8 -*-
+#!/usr/bin/python
```
Comment (luckyh) [inline virttest/shared/scripts/virtio_console_guest.py:2]: Well, IMO it has to be the first line for taking effect.

Comment (nanliu-r) [reply virttest/shared/scripts/virtio_console_guest.py]: > Well, IMO it has to be the first line for taking effect.

Updated and thanks.

**Comment by nanliu-r** [comment]
@luckyh @6-dehan Hi, I updated the commit message this time, please help update this, thanks.

**Review by 6-dehan — APPROVED** [review]

**Review by luckyh — APPROVED** [review]
LGTM

**Pull request merged**